plugin.cpp

```diff
@@ -82,7 +82,7 @@ struct AssetAction {
 typedef struct
 {
 	FledgeFilter *handle;
-	std::map<std::string, AssetAction> *assetFilterConfig;
+	std::vector<std::pair<std::string, AssetAction>> *assetFilterConfig;
```

Comment:
It will require lot of changes in the existing code.  Because after rename / split asset name will be changed.  So if I start collecting vector<AssetAction> it will not be suffice as new asset name will be required.  Let's wait for this until next JIRA for refactoring is created.

Reply:
The requirement here is to resolve a particular problem, that multiple ruels are not being executed, not completely restructure the code. We need to concentrate on the requirement as we have a customer waiting for this fix.  General style comments and possible restructing is fine, but we have to make sure we do not miss the point of why we are doing this, to fix a deficiency for a customer. Perfection is not always possible.

Reply:
@ashwini-k-pandey these refactoring points can be added to the doc you have for the Jira you had logged, among other such points and inconsistency fixes you had mentioned.

Here, We need to make sure plugin work per Jira scope, backward compatible and no regression introduced, as customer site deployment needs this as soon as possible.

We also have limited unit tests for now; considering the valid refactoring points and their need.